internal/service/advancedcluster/resource_advanced_cluster.go

```diff
@@ -576,12 +580,16 @@ func resourceRead(ctx context.Context, d *schema.ResourceData, meta any) diag.Di
 		return diag.FromErr(fmt.Errorf(ErrorClusterAdvancedSetting, "replication_specs", clusterName, err))
 	}
 
-	processArgs, _, err := connV220240530.ClustersApi.GetClusterAdvancedConfiguration(ctx, projectID, clusterName).Execute()
+	processArgs20240530, _, err := connV220240530.ClustersApi.GetClusterAdvancedConfiguration(ctx, projectID, clusterName).Execute()
+	if err != nil {
+		return diag.FromErr(fmt.Errorf(errorConfigRead, clusterName, err))
+	}
+	processArgs, _, err := connV2.ClustersApi.GetClusterAdvancedConfiguration(ctx, projectID, clusterName).Execute()
```

Comment:
Why do we need both `processArgs20240530` and `processArgs`? Will the `GetClusterAdvancedConfiguration` ever return nil?

Reply:
ChangeStreamOptionsPreAndPostImagesExpireAfterSeconds is only available in `processArgs` which is from the latest SDK versrion

Reply:
Can we not remove the `processArgs20240530`?